lib/Common/Core/ConfigParser.cpp

```diff
@@ -358,7 +358,7 @@ void ConfigParser::ParseRegistryKey(HKEY hk, CmdLineArgsParser &parser)
 
 #ifdef _WIN32
 
-void ConfigParser::SetConfigStringFromRegistry(_In_ HKEY hk, _In_ const char16* subKeyName, _In_ const char16* valName, _Inout_ Js::String& str)
```

Comment:
Does this fix 18281928? If so, why is this the fix vs just changing _In_ to _In_z_? Windows-style UPPERCASEVARIABLETYPEDEFS are something we generally have been moving away from.

Reply:
Agree -- especially if that type is not defined as per `char16` on all platforms, we really prefer the char16 for various technical reasons.

Reply:
Okay, thanks for letting me know.